src/nl/surf/eduhub_rio_mapper/rio/loader.clj

```diff
@@ -214,77 +215,112 @@
   (assert (goedgekeurd? element))                           ; should fail elsewhere with error http code otherwise
   (-> element xml-utils/element->edn json/write-str))
 
+(defn- rio-sexp-for-type [{::ooapi/keys [id]
```

Comment:
Kan je deze een duidelijker naam geven? input argument is niet `type`

Reply:
Sure